target_chains/aptos/contracts/sources/governance/governance_action.move

```diff
@@ -11,6 +11,7 @@ module pyth::governance_action {
         value: u8,
     }
 
+    #[lint::skip(unnecessary_numerical_extreme_comparison)]
```

Comment:
I think something changed with Aptos again and there's a new lint error because CONTRACT_UPGRADE=0. The comparison is intentional though, so we want to ignore.